beacon-chain/rpc/prysm/v1alpha1/debug/p2p.go

```diff
@@ -91,8 +92,12 @@ func (ds *Server) getPeer(pid peer.ID) (*ethpb.DebugPeerResponse, error) {
 	if err != nil || !ok {
 		aVersion = ""
 	}
+	sProtocols := make([]string, len(protocols))
```

Comment:
Without this, the code base won't compile in my setup.

Reply:
There's a `protocol.ConvertToStrings()`